setup.cfg

```diff
@@ -93,6 +93,8 @@ plot =
     ipython>=7.13.0,<=7.20.0
 trx =
     trx-python
+ray =
+    ray
```

Comment:
We could combine these: 

```suggestion
    ray
```

If we go with that, you would have to remove the "ray" option below on L106

Reply:
changed in last commit

Reply:
done!